src/Servers/Kestrel/Core/src/ListenOptionsHttpsExtensions.cs

```diff
@@ -258,10 +261,14 @@ public static ListenOptions UseHttps(this ListenOptions listenOptions, ServerOpt
         {
             // HttpsOptionsCallback is an internal delegate that is just the ServerOptionsSelectionCallback + a ConnectionContext parameter.
             // Given that ConnectionContext will eventually be replaced by System.Net.Connections, it doesn't make much sense to make the HttpsOptionsCallback delegate public.
```

Comment:
This comment should be updated to indicate the return type is different than the ServerOptionsSelectionCallback now too.